ex-35/index.js

```diff
@@ -0,0 +1,14 @@
+let chr = `n`;
+
+const checkChrtr = (str) =>  {
```

Comment:
Предполагается, что функция получит и символ и строку
```suggestion
const checkChrtr = (chr, str) =>  {
```